### ISSUE TITLE
How to use this repository for a cobot

### ISSUE THREAD

**Issue by SakethTecholution** [dora-rs/dora-lerobot#21]
I have a cobot made by doosan just like UR robot . And I have ROS 2 Support of it. How to use DORA along with it to train a robot. The major issue with that is they won't be having leader and follwwr configuration. So we need some kind of teleoperative interface (gamepad or keyboard) . How to make doraand act policy support for these kind of robots. If any one can suggest me a pipeline I can work on it

**Comment by haixuanTao** [comment]
Yes, so I am still iterating on the process but i have added a dora-keyboard: https://github.com/dora-rs/dora/tree/main/node-hub/dora-keyboard that makes it possible to collect keystroke into dora and then control a robot. 

I use this to then do inverse kinematics ( send cartesian position ) to the robot to make it moves.

The idea is then to collect those keyboard teleoperated data using a recorder and then generating a policy out of it,

We are still iterating on the design for ACT but it will also be a supported platform.
